src/main.py

```diff
@@ -21,12 +21,14 @@ def main():
                     level=log.INFO, stream=sys.stdout)
     spath = os.path.abspath('') + '\\sounds'
     cap = cv.VideoCapture(0)
-    pgame = Game(cap, spath)
+
     monitor = get_monitors()
     m_width = monitor[0].width
     m_height = monitor[0].height
     while cap.isOpened():
         success, img = cap.read()
+        height, width = img.shape[:2]
+        pgame = Game(height, width, spath)

```

Comment:
relocate it back, because there you create a new class object every frame

Reply:
I have a problem with video playback. the piano shrank with him. this is the only way I have found

Reply:
try to create a variable `is_settings_changed`
then type this redefinition:
```
if is_settings_changed
    pgame = Game(height, width, spath)
```
but in main.py you mast to move this string out of `while`